### ISSUE TITLE
Detect Cycles of Length 1

### ISSUE THREAD

**Issue by ChristopherChudzicki** [tmont/tarjan-graph#3]
Currently `getCycles()` simply returns strongly-connected components of length strictly greater than 1. But some strongly connected components of length 1 are cycles (for example, a vertex with itself as only child)

For example:
```js
import Graph from 'tarjan-graph'

const graph = new Graph()
graph.add('a', ['b', 'c', 'd'] )
graph.add('b', ['a'] )
graph.add('c', [])
graph.add('d', ['d'] )

const cycles = graph.getCycles()
// current behavior: returns [[ Vertex('a'), Vertex('b) ]]
// desired behavior: returns [[ Vertex('a'), Vertex('b) ], [Vertex('d')]]
// Note: Should NOT include [Vertex('c')], since it is not a cycle
```

Is this a bug, or by-design?

Thanks for a great library!

**Comment by tmont** [comment]
That seems like a useful thing to do. I'll take a look.

**Comment by tmont** [comment]
fixed in `@2.0.0`

**Comment by ChristopherChudzicki** [comment]
Awesome, thanks!
